docs/install_gui.md

```diff
@@ -0,0 +1,38 @@
+# Installing OVOS-GUI
+The GUI is a totaly optional component of OVOS, but adds a ton more functionality to the device.  Some skills will not work without one.
+
+## About the GUI
+The OVOS GUI is an independent component of OVOS which uses QT5/6 to display information on your devices screen.  It is touchscreen compliant [1](#Footnotes#1), and has an on screen keyboard for entering data.  On a Raspberry Pi, the GUI runs in a framebuffer, therefore not needing a full window manager.  This saves resources on underpowered devices.
+
+[mycroft-gui-qt5](https://github.com/OpenVoiceOS/mycroft-gui-qt5) is a fork of the origional [mycroft-gui](https://github.com/MycroftAI/mycroft-gui) which has had its licence changed to GPL v3 for all new code.
```

Comment:
note: license change was reverted in mycroft repo

Reply:
I will remove that also